signer-service/src/api/middlewares/validators.js

```diff
@@ -98,6 +98,22 @@ const validatePostSwapSubsidizationInput = (req, res, next) => {
   next();
 };
 
+const validateSep10Input = (req, res, next) => {
+  const { challengeXDR, outToken, clientPublicKey } = req.body;
+  if (!challengeXDR) {
+    return res.status(400).json({ error: 'Missing Anchor challenge' });
```

Comment:
Nit: might be nice to give the exact name of the parameter that is missing in these `400` messages. In this case, mentioning that `challengeXDR` is missing.